code/game/objects/items/books/manuals.dm

```diff
@@ -9,20 +9,44 @@
 	unique = 1
 
 
+//Engineering related manuals
+
+/obj/item/book/manual/engineering_guide
+	name = "Tools, Radiowaves And Electrical Grids"
+	desc = "A book containing basic and yet important information regarding engineering procedures."
+	icon_state = "book_engineering"
+	item_state = "book_engineering"
+	author = "Colonial Marines Engineer Association"
+	title = "Tools, Radiowaves And Electrical Grids"
+
+	dat = {"
+
+		<html><head>
+		</head>
+
+		<body>
+		<iframe width='100%' height='500px' src="https://cm-ss13.com/wiki/Guide_to_Engineering" frameborder="0" id="main_frame"></iframe> </body>
```

Comment:
```suggestion
		<iframe style="width:100%; height:85vh;" src="https://cm-ss13.com/wiki/Guide_to_Engineering" frameborder="0" id="main_frame"></iframe> </body>
```
same for rest

Reply:
![image](https://github.com/user-attachments/assets/7cde3981-a681-4dc3-9039-40a9be30f049)